source/codegen/metadata/nixnet/enums.py

```diff
@@ -87,7 +87,7 @@
             }
         ]
     },
-    'DBProperties': {
+    'DBProperty': {
```

Comment:
Both you and @ni-pkhare 's changes have these extra diffs in them. Is this going to get submitted separately?